content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/electric_dipole_definition/index.rst

```diff
@@ -0,0 +1,128 @@
+.. _definition_electric_dipole_index:
+
+Defining the Electrical Current Dipole
+======================================
+
+
+.. Purpose::
+
+    Here, we provide a physical description of the electrical current dipole.
+    This is used to develop a mathematical expression which can be used to replace the electrical source term in Maxwell's equations. 
+
+
+**General Definition**
+
+
+.. figure:: images/E_source_current_dipole.png
```

Comment:
instead of including a png figure, could we use the code? That way these figures are reproducible, and people can use the code as a starting point. For example, like the last figure on this page: http://em.geosci.xyz/content/maxwell3_fdem/circuitmodel_for_eminduction/index.html

```
.. plot::

    from em_examples.EMcircuit import Qfun
    import numpy as np
    import matplotlib.pyplot as plt
    L = 1.
    R = 2000.
    alpha = np.logspace(-3, 3, 100)
    alpha, Q = Qfun(R, L, None, alpha=alpha)
    fig = plt.figure(figsize=(5, 3))
    ax1 = plt.subplot(111)
    ax1.semilogx(alpha, Q.real, 'k', lw=3)
    ax1.semilogx(alpha, Q.imag, 'r', lw=3)
    ax1.grid(True)
    ax1.legend(("Real","Imaginary"), loc=2)
    ax1.set_xlabel("Induction number ($\\alpha$)")
    ax1.set_ylabel("Response function (Q)")
    plt.tight_layout()
    plt.show()
```
produces
![image](https://cloud.githubusercontent.com/assets/6361812/23597638/5e25c3c6-01e9-11e7-92b4-b6921cad15d9.png)